parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -138,7 +142,7 @@ createCodebaseOrError debugName path action = do
           Q.createSchema
           void . Ops.saveRootBranch v2HashHandle $ Cv.causalbranch1to2 Branch.empty
 
-      sqliteCodebase debugName path Local action >>= \case
+      sqliteCodebase debugName path Local DontMigrate action >>= \case
```

Comment:
What's the significance of this change? Were it to preserve existing behavior it'd be `MigrateAfterPrompt`, right? But was that undesirable behavior for this `createCodebaseOrError` combinator for some reason?

Reply:
I don't think the value we pass here matters at all. The requirement is that we're making a brand new codebase, so necessarily it won't require a migration.

If you'd rather I put something else here that's fine, but I don't think it'd make any observable difference unless I'm missing something.